pipeline_dp/aggregate_params.py

```diff
@@ -81,7 +78,6 @@ class AggregateParams:
     high: float = None  # deprecated
     min_value: float = None
     max_value: float = None
-    public_partitions: Any = None
```

Comment:
This is a part of the public API, it's better to help the user to fix their old code. Could you please deprecate them the same as with `high` 3 lines above.